xbmc/pvr/channels/PVRChannelGroupInternal.cpp

```diff
@@ -341,7 +341,7 @@ bool CPVRChannelGroupInternal::CreateChannelEpgs(bool bForce /* = false */)
 
   if (HasChangedChannels())
   {
-    g_EpgContainer.PersistTables();
```

Comment:
To be honest: I only wanted to get it run because kodi was not usable anymore.
PVR pumped EPG into kodi at the same time it was loaded from DB.
EPG was totally mixed, nothing was in order and it segfaulted

I didn't looked behind the scene - this ticket here cost me so much time and discussion and changes - so it was left mostly as it was before.

Yes, it seems to be a good point to remove this line, if it solves the sync issue.
Thanks, @FernetMenta